app/views/users/index.html.erb

```diff
@@ -1,2 +1,15 @@
-<h1>Users</h1>
-<p>Here is the list of users.</p>
+<% @users.each do |user| %>
+  <%= link_to user_path(user), id: "user_#{user.id}", class: "main-container" do %>
+    <div class="user-container">
+      <div class="user-photo">
+        <%= image_tag(user.photo, alt: 'User Photo', class: 'photo') %>
+      </div>
+      <div class="user-name">
+        <h2><%= user.name %></h2>
+      </div>
+      <div class="user-posts-count">
+        <p class="user-posts-count">Number of posts: <%= user.posts.count %></p>
+      </div>
+    </div>
+  <% end %>
```

Comment:
- **[OPTIONAL]** It would have been a good idea to have this in a partial and then import it across the page where it is needed instead of repeating code. Also partials help clean up the views

Reply:
Sure for DRY. Thanks for the suggestion. 😄